File: Data Structures & Algorithms/anagram-groups/submission-0.cpp
class Solution {
public:
    vector<vector<string>> groupAnagrams(vector<string>& str) 
    {
       vector<vector<string>>s;
       map<string,vector<string>>m;
       for(auto it:str)
       {
        string temp=it;
        sort(it.begin(),it.end());
        m[it].push_back(temp);
       }
        for(auto it: m)
       {
        s.push_back(it.second);
       }
       
       return s;
    }
};

File: Data Structures & Algorithms/merge-k-sorted-linked-lists/submission-0.py
# Definition for singly-linked list.
# class ListNode:
#     def __init__(self, val=0, next=None):
#         self.val = val
#         self.next = next

class Solution:    
    def mergeKLists(self, lists: List[Optional[ListNode]]) -> Optional[ListNode]:
        arr=[]
        for lst in lists:
            while lst:
                arr.append(lst.val)
                lst=lst.next
        arr.sort()
        dummy=ListNode(-1)
        temp=dummy
        for num in arr:
            newnode=ListNode(num)
            temp.next=newnode
            temp=temp.next
        return dummy.next
        